src/ssl.c

```diff
@@ -1070,6 +1115,20 @@ void ssl_setup_session_resumption(SSL_CTX **contexts, size_t num_contexts, struc
 #endif
 }
 
+void ssl_setup_session_resumption_ptls(ptls_context_t *ptls, quicly_context_t *quic)
+{
+    if (conf.ticket.update_thread != NULL) {
+        struct encrypt_ticket_ptls_t *encryptor = malloc(sizeof(*encryptor));
```

Comment:
huh, neat. I didn't realize this would work (using the variable name in its initialization), though it makes sense.